### PR TITLE
Prac 06 feedback - code review request

### PR DESCRIPTION
**Pull request by DominiqueGregoire**
@connorKeir 
Please check   formatting and naming

Thanks
Dominique

### CODE REVIEW

File: prac_06/used_cars.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"""CP1404/CP5632 Practical - Client code to use the Car class."""
+# Note that the import has a folder (module) in it.
+
+from prac_06.car import Car
+
+
+def main():
+    """Demo test code to show how to use car class.
```
Comment (connorKeir) [inline prac_06/used_cars.py:8]: "Car class"

Comment (connorKeir) [inline prac_06/used_cars.py]: Needs a capital, "How to use a Car class."

File: prac_06/guitar.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+"""CP1404
+Initialise the guitar class, fields will include name, year and cost.
+"""
+
+YEAR = 2020  # set the current year as a constant
```
Comment (connorKeir) [inline prac_06/guitar.py]: This is a class variable and should be inside the class definition.

File: prac_06/guitar.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        """return string formatting"""
+        return "{self.name} ({self.year}) : ${self.cost:.2f}".format(self=self)
+
+    def get_age(self):
+        """return the age of the guitar"""
+        return YEAR - self.year
```
Comment (connorKeir) [inline prac_06/guitar.py]: If year is made into a class variable you then reference it with "self.YEAR" . Potentially it should be a different name as it is a class variable and not a constant. (might need to be changed if the year is changed.

**Comment by connorKeir** [comment]
Pretty good.

File: prac_06/used_cars.py (hunk 1 above)
Comment (DominiqueGregoire) [reply prac_06/used_cars.py]: Cool, thanks for pointing that out. will fix it

File: prac_06/guitar.py (hunk 3 above)
Comment (DominiqueGregoire) [reply prac_06/guitar.py]: you are right. I see why it should be a class variable instead of a constant.  will amend it.

**Pull request merged**